libcontainer/container_linux.go

```diff
@@ -1054,9 +1051,6 @@ func (c *linuxContainer) isRunning() (bool, error) {
 func (c *linuxContainer) isPaused() (bool, error) {
 	data, err := ioutil.ReadFile(filepath.Join(c.cgroupManager.GetPaths()["freezer"], "freezer.state"))
 	if err != nil {
-		if os.IsNotExist(err) {
-			return false, nil
-		}
```

Comment:
iirc i think this is where we check if the host has freezer cgroup, if not the application obviously cant be frozen, hence we can resolved the pause state here.